charts/tempo/templates/statefulset.yaml

```diff
@@ -103,6 +103,18 @@ spec:
         {{- end }}
         - mountPath: /conf
           name: tempo-query-conf
+      {{- with .Values.affinity }}
+      affinity:
+        {{- tpl . $ | nindent 8 }}
```

Comment:
any particular reason you are using `tpl` on this line but `toYaml` on the others?

Reply:
My bad: I copied it from the `tempo-distributed` chart without realising why `tpl` was used in the first place. In our case, `toYaml` is sufficient, just pushed the change.